swift/common/swob.py

```diff
@@ -1183,7 +1194,10 @@ def _response_iter(self, app_iter, body):
             if self.last_modified and self.request.if_modified_since \
                and self.last_modified <= self.request.if_modified_since:
                 self.status = 304
```

Comment:
Should we make changes for all the other status numbers in that function? Coz that bare 304 was already there.